dsl/properties/scripts/ImportMicroservices.groovy

```diff
@@ -850,18 +850,17 @@ public class ImportMicroservices extends EFClient {
         report.write(text)
         String jobStepId = System.getenv('COMMANDER_JOBSTEPID')
 
-        def reportName = "Docker Import Microservices Report (${randomSuffix})"
+        def reportName = "Docker Import Microservices Report"
         publishLink(reportName, "/commander/jobSteps/${jobStepId}/${reportFilename}")
     }
 
     def publishLink(String name, String link) {
         setEFProperty("${REPORT_URL_PROPERTY}${name}", link)
         try {
-            setEFProperty("/myJob/report-urls/${name}",
-                    "<html><a href=\"${link}\" target=\"_blank\">${name}</a></html>")
+            setEFProperty("/myJob/report-urls/${name}", link)
```

Comment:
The code line above `setEFProperty("${REPORT_URL_PROPERTY}${name}", link) ` is already setting the report url for the job.
1. So why do we need to set it again here?
2. Why is this in a try-catch block? Under what condition do we expect an error here?

Reply:
We need to also update the plugin version in plugin.xml (see the JIRA for the version number) since this is the first fix since we released the the last version.
And add an entry in the release notes for this version and the issue being addressed.